todo/src/components/Todo.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+class Todo extends Component {
+  render() {
+    return (
+      <li
+        style={props.todo.completed ? { textDecoration: 'line-through' } : null}
+        onClick={() => {
+          props.toggleCompleted(props.todo.id);
+        }}
+        id={props.todo.id}
+      >
+        {props.todo.value}
+      </li>
+    );
+  }
+}
+const mapStateToProps = state => {
+  return {
+    ...props
```

Comment:
I was wrong about spreading in props here. We actually don't need the `mapStateToProps` function here since we are passing in each todo from `Todos.js`. You just need to make sure that `todo` is an object with `value`, and `completed` properties in order to use them in this component.